packages/opentelemetry-exporter-jaeger/test/jaeger.test.ts

```diff
@@ -150,5 +151,17 @@ describe('JaegerExporter', () => {
         assert.strictEqual(result, ExportResult.SUCCESS);
       });
     });
+
+    it('should use httpSender if config.endpoint is setten and set x-opentelemetry-outgoing-request header', () => {
```

Comment:
what is `x-opentelemetry-outgoing-request`, why does it need to be sent to Jaeger?

Reply:
See https://github.com/open-telemetry/opentelemetry-js/pull/965#discussion_r407714610 but it's a header we use internaly to avoid tracing outgoing requests done by exporters hence creating recursion (span created -> exported to http -> span created for the export -> etc)

Reply:
It is not actually sent to jaeger. The http instrumentation sees it, does not trace the request, and strips it from the request.

Reply:
1. this explanation should be added to the constant that defines this header
2. wherever the telemetry is being sent, it may be causing issues like this one https://github.com/openzipkin/zipkin-support/issues/6
3. considering that JS objects can be easily monkey patched, isn't there another way to tag the requests besides using the http header?

Reply:
@dyladan It seems we don't actually remove it before sending https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-plugin-http/src/utils.ts#L303, https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-plugin-http/src/http.ts#L384), we should fix that in a separate PR.

@yurishkuro For your 3rd point, i think a header is simpler to maintain across multiples exporters. Using a custom hidden property would produce typescript type errors that we'll need to ignore manually, which is something we want to avoid

Reply:
@vmarchaud as long as you don't send the header out. But I still think a hidden property is safer. Access to that property can be encapsulated into a pair of helper methods, to deal with type safety.

Reply:
The spec is working on a way to do this with a context property anyways so whatever we pick here will be superseded by that eventually anyways.

Reply:
Should I refactor it?

Reply:
@leonardodalcin I don't think so, we don't have an implementation yet so i think the way to go would be to merge this and implement the new system in a follow up PR if you want to